client/modules/User/components/APIKeyForm.jsx

```diff
@@ -39,7 +39,8 @@ class APIKeyForm extends React.Component {
   }
 
   removeKey(key) {
-    const message = `Are you sure you want to delete "${key.label}"?`;
+    // const message = `Are you sure you want to delete "${}"?`;
+    const message = i18next.t('APIKeyForm.ConfirmDelete', { key_label: key.label });
```

Comment:
This should use `withTranslation`, not `i18next` as it's within a component.

Reply:
Ok, the change requested it is complete, can you check this please?

Reply:
You also need to remove the `i18next` import from the top of the file as it's no longer used. The lint should be warning about this.

Reply:
Ok, the import is deleted, can you have a look please.?